### PR TITLE
[Workers] Add Workers changelog through 8/8/24, update supported node:crypto APIs

### PR DESCRIPTION
**Pull request by fhanau**
### Summary

All of these are prior changes that were missing so far. Also includes some formatting improvements and updates the list of supported APIs for node:crypto.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: content/workers/runtime-apis/nodejs/crypto.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The [WebCrypto API](/workers/runtime-apis/web-crypto/) is also available within
 
 | API | Supported? | Notes |
 |-----|------------|-------|
-| [Certificate](https://nodejs.org/api/crypto.html#class-certificate) |  | |
+| [Certificate](https://nodejs.org/api/crypto.html#class-certificate) | ✅ | |
```
Comment (fhanau) [inline content/workers/runtime-apis/nodejs/crypto.md]: CC @jasnell this adds documentation for newly supported APIs

File: data/changelogs/workers.yaml [hunk 2]
```diff
@@ -7,15 +7,29 @@ productAreaLink: /workers/platform/changelog/platform/
...
+    - Workers now support the standard [EventSource](/workers/runtime-apis/eventsource/) API.
+    - Fixed a bug where when writing to an HTTP Response body would sometimes hang when the client disconnected (and sometimes throw an exception). It will now always throw an exception.
 - publish_date: '2024-07-01'
   description: |-
     - When using [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/), you can now use [version overrides](/workers/configuration/versions-and-deployments/gradual-deployments/#version-overrides) to send a request to a specific version of your Worker.
+- publish_date: '2024-06-28'
+  description: |-
+    - Fixed a bug which caused `Date.now()` to return skewed results if called before the first I/O of the first request after a Worker first started up. The value returned would be offset backwards by the amount of CPU time spent starting the Worker (compiling and running global scope), making it seem like the first I/O (e.g. first fetch()) was slower than it really was. This skew had nothing to do with Spectre mitigations; it was simply a longstanding bug.
```
Comment (fhanau) [inline data/changelogs/workers.yaml]: I edited the internally specified release note a bit here, but it's still quite wordy. Open to suggestions on making it more concise.

**Review by irvinebroque — APPROVED** [review]

File: src/content/changelogs/workers.yaml [hunk 3]
```diff
@@ -29,12 +43,13 @@ entries:
   description: |-
     - When using [`nodejs_compat` compatibility flag](/workers/runtime-apis/nodejs/), the `buffer` module now has an implementation of `isAscii()` and `isUtf8()` methods.
     - Fixed a bug where exceptions propagated from [JS RPC](/workers/runtime-apis/rpc) calls to Durable Objects would lack the `.remote` property that exceptions from `fetch()` calls to Durable Objects have.
+    - The `cloudflare:workers` module now exports a new `compatFlags` API that can be used to determine if a given compatibility flag is enabled for the worker.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml]: This was reverted

File: src/content/changelogs/workers.yaml [hunk 4]
```diff
@@ -10,15 +10,29 @@ entries:
...
+    - Workers now support the standard [EventSource](/workers/runtime-apis/eventsource/) API.
+    - Fixed a bug where when writing to an HTTP Response body would sometimes hang when the client disconnected (and sometimes throw an exception). It will now always throw an exception.
 - publish_date: '2024-07-01'
   description: |-
     - When using [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/), you can now use [version overrides](/workers/configuration/versions-and-deployments/gradual-deployments/#version-overrides) to send a request to a specific version of your Worker.
+- publish_date: '2024-06-28'
+  description: |-
+    - Fixed a bug which caused `Date.now()` to return skewed results if called before the first I/O of the first request after a Worker first started up. The value returned would be offset backwards by the amount of CPU time spent starting the Worker (compiling and running global scope), making it seem like the first I/O (e.g. first fetch()) was slower than it really was. This skew had nothing to do with Spectre mitigations; it was simply a longstanding bug.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml:42]: @mikenomitch maybe a better way to explain

File: src/content/changelogs/workers.yaml [hunk 5]
```diff
@@ -10,15 +10,29 @@ entries:
     -  Workers now support the [`allow_custom_ports` compatibility flag](/workers/configuration/compatibility-dates/#allow-specifying-a-custom-port-when-making-a-subrequest-with-the-fetch-api) which enables using the `fetch()` calls to custom ports.
 - publish_date: '2024-07-19'
   description: |-
-    -  Workers with the [mTLS](/workers/runtime-apis/bindings/mtls/) binding now support [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/).
+    - Workers with the [mTLS](/workers/runtime-apis/bindings/mtls/) binding now support [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/).
+- publish_date: '2024-07-18'
+  description: |-
+    - Added a new `truncated` flag to tail worker events to indicate when the event buffer is full and events are being dropped.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml]:
```suggestion
    - Added a new `truncated` flag to [Tail Worker](/workers/observability/logging/tail-workers/) events to indicate when the event buffer is full and events are being dropped.
```

Doesn't look like this was documented here?

https://developers.cloudflare.com/workers/observability/logging/tail-workers/

Can we tag author of the change here and make sure we follow through on it? @rohinlohe

**Review by irvinebroque — COMMENTED** [review]
Thanks for follow through on missing items @fhanau

**Comment by fhanau** [comment]
Updated this to add two more public-facing changes added since.

File: src/content/changelogs/workers.yaml [hunk 6]
```diff
@@ -8,17 +8,37 @@ entries:
 - publish_date: '2024-08-19'
   description: |-
     -  Workers now support the [`allow_custom_ports` compatibility flag](/workers/configuration/compatibility-dates/#allow-specifying-a-custom-port-when-making-a-subrequest-with-the-fetch-api) which enables using the `fetch()` calls to custom ports.
+- publish_date: '2024-08-17'
+  description: |-
+    -  Updated v8 to version 12.8.
+- publish_date: '2024-08-15'
+  description: |-
+    -  The `internal_writable_stream_abort_clears_queue` compatibility flag will ensure that certain WritableStream abort() operations are handled immediately rather than lazily, ensuring that the stream is appropriately aborted when the consumer of the stream is no longer active.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml]: Can we link to docs for this flag?

Comment (fhanau) [reply src/content/changelogs/workers.yaml]: Looks like the flag is not documented so far – @jasnell  can you make a PR for this?

**Review by irvinebroque — APPROVED** [review]
We can merge this but should follow up to make sure that compat flag is documented. @jasnell

**Pull request merged**